### PR TITLE
feat: Arbitrum FQT Updates

### PR DESCRIPTION
**Pull request by dextracker**
# Description

bumped packages versions for new fill Quote transformer @ `0x542c7dfc7959d8d03c5b738478213885ab3f214e`

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [ ] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [ ] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

File: src/asset-swapper/utils/market_operation_utils/types.ts [hunk 1]
```diff
@@ -301,6 +303,8 @@ export interface FinalUniswapV3FillData extends Omit<UniswapV3FillData, 'pathAmo
     // The uniswap-encoded path that can fll the maximum input amount.
     uniswapPath: string;
     gasUsed: number;
+    //only used for gas estimation
```
Comment (kyu-c) [inline src/asset-swapper/utils/market_operation_utils/types.ts:306]: looks like this was added by a mistake?

**Review by kyu-c — APPROVED** [review]
LGTM -- one nit

File: src/asset-swapper/utils/market_operation_utils/types.ts (hunk 1 above)
Comment (dextracker) [reply src/asset-swapper/utils/market_operation_utils/types.ts:306]: i actually need this for the types :(

Comment (dextracker) [reply src/asset-swapper/utils/market_operation_utils/types.ts:306]: nvm actually that one i dont need

**Pull request merged**